### PR TITLE
Allow `tmt try IP@connect`

### PR DESCRIPTION
**Pull request by lukaszachy**
Pull Request Checklist

* [x] implement the feature
* [x] adjust plugin docstring

### CODE REVIEW

**Comment by lukaszachy** [comment]
@psss how should be this documented? Having to change all `IMAGE@HOW` into `IMAGE_OR_IP@HOW` doesn't seem a good idea to me as it will just be confusing.

**Review by KwisatzHaderach — APPROVED** [review]

**Review by janhavlin — APPROVED** [review]

File: tmt/cli.py [hunk 1]
```diff
@@ -1666,6 +1666,11 @@ def try_command(context: Context, image_and_how: str, **kwargs: Any) -> None:
     \b
         tmt try fedora@container
         tmt try centos-stream-9@virtual
+
+    Or connect to the running guest by specifying FQDN or IP address:
```
Comment (psss) [inline tmt/cli.py:1670]:
```suggestion
    Or connect to the running guest by specifying hostname or IP address:
```

Let's use rather `hostname` than the abbreviation?

**Review by psss — CHANGES_REQUESTED** [review]
Looks good, I'd just avoid the abbreviation.

**Review by therazix — APPROVED** [review]

File: tmt/cli.py [hunk 2]
```diff
@@ -1684,6 +1689,13 @@ def try_command(context: Context, image_and_how: str, **kwargs: Any) -> None:
         else:
             options = {"image": image_and_how}
 
+        # For 'connect' rename 'image' to 'guest'
+        try:
+            if options['how'] == 'connect':
+                options['guest'] = options.pop('image')
+        except KeyError:
+            pass
```
Comment (thrix) [inline tmt/cli.py:1697]: with `get` we can avoid the try / except handling?

```suggestion
            if options.get('how') == 'connect':
                options['guest'] = options.pop('image')
```

**Review by martinhoyer — APPROVED** [review]
lgtm, works as expected.

File: tmt/cli.py (hunk 1 above)
Comment (lukaszachy) [reply tmt/cli.py:1670]: it's not the same, is it? https://serverfault.com/questions/269838/what-is-the-difference-between-a-hostname-and-a-fully-qualified-domain-name

Comment (martinhoyer) [reply tmt/cli.py:1670]: Agreed to keep FQDN on hacking session.

**Comment by happz** [comment]
Unrelated failures, merging.

**Pull request merged**